pkg/graph/resolvers/system_intake_system_test.go

```diff
@@ -83,3 +83,104 @@ func (s *ResolverSuite) TestIntakeRelatedSystems() {
 		s.Empty(data)
 	})
 }
+
+func (s *ResolverSuite) TestSystemIntakesByCedarSystemID() {
+	ctx := s.testConfigs.Context
+
+	var (
+		open1 uuid.UUID
+		open2 uuid.UUID
+
+		closed uuid.UUID
+	)
+
+	const (
+		system1 = "1"
+		system2 = "2"
+		system3 = "3"
+		system4 = "4"
+	)
+
+	s.Run("test getting system intakes by cedar system id", func() {
+		// create some intakes
+		intake1 := models.SystemIntake{
+			EUAUserID:   testhelpers.RandomEUAIDNull(),
+			RequestType: models.SystemIntakeRequestTypeNEW,
+			State:       models.SystemIntakeStateOpen,
+		}
```

Comment:
This code is my fault, but there's a [helper](https://github.com/CMS-Enterprise/easi-app/blob/main/pkg/graph/resolvers/resolver_test.go#L189-L198) you can use to shorten this to `intake1 := s.createNewIntake()`

Reply:
i need to create a closed one though